openshift/odo
# [WIP] Modifies push to push only changed files during execution

fixes #2018 

How to test:

1. create a component and push
2. create a new file or modify a file and push
3. check inside the container that only the modified/added files were pushed by using `stat` and checking the timestamps
